Sources/FluentSQL/SQLQueryConverter.swift

```diff
@@ -210,11 +210,11 @@ public struct SQLQueryConverter {
                 let right: SQLExpression
                 switch method {
                 case .anywhere:
-                    right = SQLRaw("%" + string.description + "%")
+                    right = SQLRaw("\"%" + string.description + "%\"")
```

Comment:
Using `SQLLiteral.string` here instead of `SQLRaw` will add quotes appropriate for each DB.

Reply:
perfect, that's what I was looking for!

Reply:
Per discussion in Discord, use `right = SQLBind("%\(string)%")`